src/NuGet.Core/NuGet.Protocol/PublicAPI/net472/PublicAPI.Shipped.txt

```diff
@@ -14,7 +14,6 @@ NuGet.Protocol.AutoCompleteResourceV2Feed
 NuGet.Protocol.AutoCompleteResourceV2FeedProvider
 NuGet.Protocol.AutoCompleteResourceV2FeedProvider.AutoCompleteResourceV2FeedProvider() -> void
 NuGet.Protocol.AutoCompleteResourceV3
-~NuGet.Protocol.AutoCompleteResourceV3.AutoCompleteResourceV3(NuGet.Protocol.HttpSource client, NuGet.Protocol.ServiceIndexResourceV3 serviceIndex, NuGet.Protocol.RegistrationResourceV3 regResource) -> void
```

Comment:
Don't break existing APIs unless it's really needed. Instead, add a new overload, and have the old overload call the new one with a "sensible" default for the new parameters.

Mark the old method as obsolete if there's a good enough justification, or you can use the banned API analyzer to just ban the old API within our own repo, without affecting custoemrs who might be calling it in their own apps.